packages/jsii-spec/lib/spec.ts

```diff
@@ -7,7 +7,7 @@ export interface Assembly extends Documentable {
     /**
      * The version of the spec schema
      */
-    schema: SchemaVersion.V1_0;
+    schema: SchemaVersion.LATEST;
```

Comment:
I think if you're going to do this, it needs a version check in the jsii-kernel and jsii-reflect as well.

Reply:
Both validate the schema, so the version check is in there.